templates/cadastre.parcel.mst

```diff
@@ -1,5 +1,5 @@
 {{#features}}
-	<li class="item" style="width:238px;">
+	<li id="{{feature_id}}" class="item" style="width:238px;">
```

Comment:
Needs explain (see previous comment) about `<li id="{{feature_id}}"`.
Thanks :)

Reply:
See above. I renamed it to `feature_ol_uid` and added another phrase to the doc.